python/ray/util/metrics.py

```diff
@@ -247,6 +247,10 @@ def __init__(
                 "the Histogram class. e.g., "
                 'Histogram("name", boundaries=[1.0, 2.0])'
             )
+        for boundary in boundaries:
+            if boundary <= 0:
+                raise ValueError("boundaries arguments should be positive values")
```

Comment:
Can you update the error message? Following this guideline; https://spark.apache.org/error-message-guidelines.html

```
for i, boundary in enumerate(boundaries):
    if boundary <= 0:
        raise ValueError(`Invalid `boundaries` argument at index {i}, {boundaries}. Use positive values for the arguments.)
```